src/components/action-pad/action-pad.tsx

```diff
@@ -207,6 +219,26 @@ export class ActionPad
     this.expandToggleEl = el;
   };
 
+  updateGroups = (): void => {
```

Comment:
I think this can be a class method and not need to store a bound function. Applies to similar functions.